graphrag/index/cli.py

```diff
@@ -108,7 +108,7 @@ def index_cli(
             ValueError("ContextOperation is invalid: It should be Active or DeActive")
         graphrag_config = _read_config_parameters(root, config, progress_reporter)
         _switch_context(graphrag_config, context_operation, context_id, progress_reporter)
-        sys.exit(0)
+        #sys.exit(0)
```

Comment:
looks unintentional.

Reply:
I see, it was. I commented this to test the whole workflow of activating the context and indexing immediately after that. However, if the expected behavior is to do this in to separate steps, I will keep this line uncommented